tests/integration/targets/elb_target/runme.sh

```diff
@@ -2,12 +2,5 @@
 
 set -eux
 
-# Test graceful failure for older versions of botocore
-source virtualenv.sh
-pip install 'botocore<=1.7.1' boto3
-ansible-playbook -i ../../inventory -v playbooks/version_fail.yml "$@"
-
 # Run full test suite
-source virtualenv.sh
-pip install 'botocore' 'boto3>=1.16.57'
```

Comment:
`boto3 >= 1.13.0` 

boto3 1.16.57 is only around 6 months old...

Reply:
Oh, I went to fast on this one.